definitions/m5/m5.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: m5-release-pipeline
+  namespace: managed
+spec:
+  params:
+    - name: mapping-file
+      type: string
+      description: mapping to apply to snapshot
+  tasks:
+    - name: get-mapping-file
+      taskRef:
+        name: get-mapping-file
+      params:
+        - name: mapping-file
+          value: "$(params.mapping-file)"
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-mapping-file
+  namespace: managed
+spec:
+  params:
+    - name: mapping-file
+      type: string
+      description: the file that defines the mapping to apply to the snapshot
+  results:
+    - name: mapping
+      description: The retrieved mapping file as a string
+  steps:
+    - name: get-mapping-file
+      image: quay.io/fedora/fedora:36
+      script: |
+        #!/bin/bash
+        #Download yq
+        curl -O -L https://github.com/mikefarah/yq/releases/download/v4.25.1/yq_linux_amd64
+        chmod +x yq_linux_amd64
+        cp yq_linux_amd64 /usr/bin/yq
+
+        mapping=$(curl $(params.mapping-file) | yq -o=json -I=0)
```

Comment:
We haven't defined the file yet. Do we(you) prefer json?

Reply:
Also, not a fan of this method. The repository would have to be public and I guess that we don't want that. We should be using a service account in the PipelineRun to be able to clone the repo. Check [this page](https://tekton.dev/vault/pipelines-v0.14.3/auth/). If we don't want to close, we should take a look to [something like this](https://stackoverflow.com/questions/18126559/how-can-i-download-a-single-raw-file-from-a-private-github-repo-using-the-comman).

Reply:
what alternative are you suggesting to json? json neatly fits a string onto one line and is easy to work with in bash and python

The service account stuff is an entirely separate ticket. I would expect that the credentials part is added to this pipeline when someone works on that ticket. So you want me to change this to `git clone` instead? Parsing a url to find the repo to clone and the path to the file then `cat`ing the file to a variable is certainly more steps than `curl`ing it

Reply:
To the second part, leave it like this and we take care of it later.

To the first part, I'm talking about the yaml to json part. If we define a json initially the validation and actually parsing would be easier. What's your opinion on this?